### DISCUSSION TITLE
Missing ROI

### DISCUSSION THREAD

**Discussion by l-espana** [ExploreASL/ExploreASL#1544] [Q&A]
Hello,

We recently noticed after running the stats on our data with multiple atlases that there are several ROI missing output. We're not really sure why this is the case after looking at the data...the masks all look good and the data has full coverage. We're specifically missing areas such as the pallidum (despite having other subcortical ROI with data). Any ideas on where this issue could be occurring? Has anyone else experienced this?

**Comment by MDijsselhof** [comment]
Hi,

Apologies for the late response. One possible reason the ROIs are missing output is because of partial volume effects and thresholding of segmentations. Could you explain what data you are using, are you not seeing results of the ROI in the columns or are they providing n/a, which template you are using and what your research questions are? Then we can tailor the help to your needs! 
Regarding the pallidum, even though this is not a small volume, the elongation might result in a lot of voxels being mixed with WM signal and subsequent thresholding that is performed, to make sure we only use voxels in which we are certain there is a large amount of GM, might result in no values obtained in that region. There are ways to circumvent this, so please let us know if we can help further!

Kind regards,

Mathijs
